src/screens/Account/AccountHeaderRight.js

```diff
@@ -39,6 +45,15 @@ const AccountHeaderRight = ({ navigation, account, parentAccount }: Props) => {
   if (!account) return null;
 
   if (account.type === "TokenAccount" && parentAccount) {
+    const mainAccount = getMainAccount(account, parentAccount);
+    const explorerView = getDefaultExplorerView(mainAccount.currency);
+
+    const url = getAccountContractExplorer(
+      explorerView,
+      account,
+      parentAccount,
+    );
+
     return (
       <>
         <Touchable event="ShowContractAddress" onPress={toggleModal}>
```

Comment:
mmh, i might be wrong, but shoudln't we return null completely if there is no url? that's what i meant by "not showing it at all if it's not possible"

Reply:
we have a modal that asks to hide the token as well we have to be able to hide the token except the modal showing the contract address link won't show :+1:

Reply:
ok